servers/lib/src/main.ts

```diff
@@ -1,11 +1,23 @@
-import { ConfigService } from "@nestjs/config";
-import { NestFactory } from "@nestjs/core";
-import AppModule from "./app.module";
-
-async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
-  const configService = app.get(ConfigService);
-  const port = configService.get<number>("PORT");
-  await app.listen(port);
+#!/usr/bin/env node
+import { Command } from "commander";
+import bootstrap from "./bootstrap";
+
+type ProgramOptions = {
+  config?: string
 }
-bootstrap();
+
+const program = new Command();
+
+program
```

Comment:
```sh
yarn build
node ./dist/src/main.js```

does not produce the expected result. The CLI options are not showing up.

Reply:
The library [https://github.com/tj/commander.js#automated-help](commander) has an automated help function (more information in the link).
I assumed that you wanted to run it like "libms -h" or "libms -c <config>".
The CLI, if no option is provided, will assume the default path is ".env".
You can use node `./dist/src/main.js -h` to see the options. 
I'm assuming that it did run the application?

Reply:
This makes sense

Reply:
How easy is it to show the same result for two commands:

```
libms
libms -h
```

Reply:
It's shouldn't be that difficult.. However, I would argue against this since it can run the app when you don't specify any input.. Because -c is not necessary for the app to work, since it will use .env as a default.

Reply:
> How easy is it to show the same result for two commands:
> 
> ```
> libms
> libms -h
> ```

Just to clarify..
You want the `-c <path>` to be required?
Because making `libms` and `libms -h` the same will cause the program to only be run when `-c <path>` is specified-

Reply:
> It's shouldn't be that difficult.. However, I would argue against this since it can run the app when you don't specify any input.. Because -c is not necessary for the app to work, since it will use .env as a default.

You are right. Let us keep it the way you suggested. If no flag is provided, `libms` uses the `.env` of the working directory and starts.

If `.env` does not exist, it will display same output as `libms -h`.

Reply:
I agree with your statement that `-c` flag should be optional. It should not be made mandatory.

My only question is the consequence of using libms without `-c` flag if there is no `.env` in the working directory.

Reply:
The new changes that has been made, will display the error message from dotenv (I think this is relevant, so that the user can know what is wrong) and the help text from -h.